pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -84,6 +84,11 @@ stdenv.mkDerivation rec {
       stripLen = 1;
       extraPrefix = "slirp/";
     })
+    (fetchpatch {
```

Comment:
do you mind adding a reason, and timeline for removing the patch?
```suggestion
    # fixes x,y,z . Can be removed next minor/patch.
    (fetchpatch {
```

Reply:
i guess the patch name mentions the CVE

Reply:
Oops, overlooked your comment. Patch will be obsolete in 4.2.1, just FTR.